components/CandidateLink/CandidateLink.tsx

```diff
@@ -11,7 +11,7 @@ interface CandidateLinkProps {
 
 export function CandidateLink(props: CandidateLinkProps): JSX.Element {
 
-    const ref: string = "/candidates/" + props.id;
+    const ref: string = "/testsubmissions/" + props.id;
```

Comment:
Could we go back to candidates?

I don't have a very strong opinion, but I think this page is really just the details for that candiate.
ie we have `/candiates` which lists all the candidates
and then `/candidates/{candidateId}` which gives details about that specific candidate (including but not limited to their test submissions)

Reply:
yes, this just means that there will slightly confusingly be a "candidates" folder with [testsubmissions].tsx inside and a candidates.tsx page (which is for listing all of the candidates) outside of this folder. If this is ok I am happy to change 👍

Reply:
you are right - slightly confusing.

I think I'd be tempted to change it to
```txt
- pages
    - candidates
        - index.tsx (will render at `/candidates`
        - [candidateId].tsx (will render at `/candidates/{candidateId}`
```

(but having `candidates.txt` outside of the `candidates` directory is also valid Next folder structure)

Reply:
oops just saw this, will update tonight to this.

Reply:
so updated to -candidates -[candidateId].tsx and left candidates.tsx outside of the candidates directory for now.